packages/test/test-end-to-end-tests/src/test/gc/gcTombstone.spec.ts

```diff
@@ -115,198 +118,316 @@ describeNoCompat("GC DataStore Tombstoned When It Is Sweep Ready", (getTestObjec
         {
             eventName: "fluid:telemetry:Container:ContainerClose",
             error: "Context is tombstoned: Call site -  process!",
             errorType: "dataCorruptionError",
         },
     ],
     async () => {
-        const { testDataObjectId, summaryVersion } = await getUnreferencedDataStoreIdAndSummaryVersion();
-
-        // Wait a sweep worthy amount of time (all containers should have closed by now)
-        await delay(sweepTimeoutMs);
-
-        // Load a new container and summarizer based on the latest summary, summarize
         const {
-            container: summarizingContainer2,
-            summarizer: summarizer2,
-        } = await loadSummarizerAndContainer(summaryVersion);
+            unreferencedId,
+            summarizingContainer,
+            summarizer,
+            summaryVersion,
+        } = await summarizationWithUnreferencedDataStoreAfterTime(sweepTimeoutMs);
         // Setup close validation
         let closeError: IErrorBase | undefined;
-        summarizingContainer2.on("closed", (error) => {
+        summarizingContainer.on("closed", (error) => {
             closeError = error;
         });
 
         // The datastore should be tombstoned now
-        await summarize(summarizer2);
+        await summarize(summarizer);
 
         // We load this container from a summary that had not yet tombstoned the datastore so that the datastore loads.
-        const container2 = await provider.loadTestContainer(testContainerConfig, { summaryVersion });
+        const container = await provider.loadTestContainer(testContainerConfig, { summaryVersion });
         // Use the request pattern to get the testDataObject - this is unsafe and no one should do this in their
         // production application
         // This does not cause a sweep ready changed error as the container has loaded from a summary before sweep
         // ready was set
-        const testDataObject2 = await requestFluidObject<ITestDataObject>(container2, testDataObjectId);
+        const dataObject = await requestFluidObject<ITestDataObject>(container, unreferencedId);
 
         // Receive an op - the summarizing container does not log a sweep ready changed error as it closes before
         // the op is processed. The summarizing container does log a sweep ready loaded error and then it should
         // process the op which causes the container to close.
-        testDataObject2._root.set("receive", "op");
+        dataObject._root.set("send an op to be received", "op");
         await provider.ensureSynchronized();
-        assert(summarizingContainer2.closed === true, `Summarizing container should close.`);
+        assert(summarizingContainer.closed === true, `Summarizing container should close.`);
         assert(closeError !== undefined, `Expecting an error!`);
         assert(closeError.errorType === "dataCorruptionError");
         assert(closeError.message === "Context is tombstoned: Call site -  process!");
```

Comment:
This error message is hard for me to read.  Can we try this instead?

```
`Context is tombstoned! Call site [${callSite}]`
```

There are some undocumented punctuation preferences I have too - no `:` (reserved for delineating an inner error message) and put `[]` around variable bits.  They're undocumented because I'm not sure they "work" as well as I hope, but trying them out when I can.